apps/ledger-live-mobile/src/components/RootNavigator/types/PtxLiveAppNavigator.ts

```diff
@@ -0,0 +1,25 @@
+import { ScreenName } from "~/const";
+
+type CommonParams = {
+  goToURL?: string;
+  lastScreen?: string;
+  name?: string;
+  platform?: string;
+  referrer?: string;
+};
+
+type ExchangeParams = CommonParams & {
+  account?: string;
+  currency?: string; // Used for the deeplink only
+  defaultAccountId?: string;
+  defaultCurrencyId?: string;
+  defaultTicker?: string;
+};
```

Comment:
Should everything here be optional?

Reply:
Yes, I think we do not have any of this param when you click directly "buy"